backend/src/utils/resourceUtils.ts

```diff
@@ -269,6 +292,14 @@ export const initializeWatchedResources = (fastify: KubeFastifyInstance): void =
   consoleLinksWatcher = new ResourceWatcher<ConsoleLinkKind>(fastify, fetchConsoleLinks);
 };
 
+export const getDashboardConfig = (): DashboardConfig => {
+  const config = dashboardConfigWatcher.getResources()?.[0] ?? DEFAULT_DASHBOARD_CONFIG;
+  return {
+    enablement: (config.data?.enablement ?? '').toLowerCase() !== 'false',
+    disableInfo: (config.data?.disableInfo ?? '').toLowerCase() === 'true',
```

Comment:
optional
if you haven't already tried it, you maaaaaay be able to just write these as. I think there is a scenario where you don't want to use this, but it currently escapes me
```
... config.data?.enablement?.toLowerCase() !==  'false',
... config.data?.disableInfo?.toLowerCase() ...
```